### DISCUSSION TITLE
Code Improvements

### DISCUSSION THREAD

**Discussion by lifof** [cagnulein/qdomyos-zwift#339] [Ideas]
Hello @cagnulein 

Do you mind if I commit some tweaks to the code, as in using QStringLiteral to avoid QString allocation ( great benefit in performance ) as a first step then using tools like Clazy to review cpp code and clang-format to format all source files.

**Comment by cagnulein** [comment]
absolutely! Submit all what you think that could improve the code and the performances!

**Reply by lifof** [reply]
Ok thanks, so I'll start working on it.

Have a great day
